Services/Administration/OJS.Services.Administration.Business/Implementations/RecurringBackgroundJobsBusinessService.cs

```diff
@@ -6,9 +6,15 @@
     public class RecurringBackgroundJobsBusinessService : IRecurringBackgroundJobsBusinessService
     {
         private readonly ISubmissionsForProcessingBusinessService submissionsForProcessing;
+        private readonly OJS.Services.Ui.Business.IParticipantsBusinessService participantsBusinessService;
 
-        public RecurringBackgroundJobsBusinessService(ISubmissionsForProcessingBusinessService submissionsForProcessing)
-            => this.submissionsForProcessing = submissionsForProcessing;
+        public RecurringBackgroundJobsBusinessService(
+            ISubmissionsForProcessingBusinessService submissionsForProcessing,
+            OJS.Services.Ui.Business.IParticipantsBusinessService participantsBusinessService)
```

Comment:
Administration should not reference Ui. Move logic into Administration's `IParticipantsBusinessService`.